source/Octopus.Tentacle.Kubernetes.Tests.Integration/Setup/agent-values.yaml

```diff
@@ -8,6 +8,17 @@ agent:
   targetEnvironments: ["development"]
   targetRoles: ["testing-cluster"]
   
+  image:
+    repository: docker.packages.octopushq.com/octopusdeploy/kubernetes-agent-tentacle
```

Comment:
Do you need the `/octopusdeploy` bit?

Reply:
yes, I believe we do...

https://github.com/OctopusDeploy/OctopusTentacle/blob/main/source/Octopus.Tentacle.Kubernetes.Tests.Integration/KubernetesClusterOneTimeSetUp.cs#L33

Reply:
Ah because we are using Artifactory as a dockerhub cache I guess?

I suppose if it doesn't work the tests will fail so I'll approve 👍